README.md

```diff
@@ -177,17 +177,28 @@ the JS file method mentioned above then you can use functions normally.
 | Target Name | Description | Required Config |
 |-------------|-------------|-----------------|
 | github      | Makes a new release for the current version (if required) and uploads the make artifacts as release assets | `process.env.GITHUB_TOKEN` - A personal access token with access to your releases <br />`forge.github_repository.owner` - The owner of the GitHub repository<br />`forge.github_repository.name` - The name of the GitHub repository <br />`forge.github_repository.draft` - Create the release as a draft, defaults to `true` <br />`forge.github_repository.prerelease` - Identify the release as a prerelease, defaults to `false` |
+| Amazon S3   | Uploads your artifacts to the given S3 bucket | `process.env.S3_SECRET` - Your secret access token for your AWS account<br />`forge.s3.accessKey` - You access key for your AWS account<br />`forge.s3.bucket` - The name of the S3 bucket to upload to<br />`forge.s3.folder` - The folder path to upload to inside your bucket, defaults to your application version<br />`forge.s3.public` - Whether to make the S3 upload public, defaults to `false`
 
 For example:
 
 ```javascript
+// github
 {
   // Assume the GitHub repository is at https://github.com/username/repo
   "github_repository": {
     "owner": "username",
     "name": "repo"
   }
 }
+
+// s3
+{
+  "s3": {
+    "accessKey": "<AWS_ACCESS_KEY>",
+    "bucket": "my_bucket_name",
+    "public": true
+  }
```

Comment:
These docs will be superseded by the site when I get the config docs there in a good state.  For now I think clear separation is better?